Microsoft.Research/Contracts/MsCorlib/System.Runtime.CompilerServices.ExtensionAttribute.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace System.Runtime.CompilerServices
 {
+#if NETFRAMEWORK_4_5
```

Comment:
:bulb: If you move this up to above the `using System;`, it won't be removed if someone does Remove and Sort Usings without this directive defined. The `#endif` should be moved to the end of the file at the same time.